app/mailers/user_notifier_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class UserNotifierMailer < ApplicationMailer
+  default :from => 'smtp.sendgrid.net'
+
```

Comment:
Since we created a google account for YouTube we could technically use that email as the sender I bet!

Reply:
Yeah good idea, Steven